runtime/src/comm/ugni/comm-ugni.c

```diff
@@ -993,10 +993,10 @@ typedef enum {
   put_64,
   get_32,
   get_64,
-  xchg_32,
-  xchg_64,
-  cmpxchg_32,
-  cmpxchg_64,
+  swap_32,
+  swap_64,
+  cmpswap_32,
```

Comment:
Nit: maybe "cswap" instead of "cmpswap", so as to match other usage (both uGNI/Aries and libfabric op names, for example).

Reply:
I'm fine with that. I stuck with cmpswap only because it was the same number of characters.